server.go

```diff
@@ -604,13 +606,18 @@ func (s *Server) appendAddrs(list []dns.RR, ifIndex int) []dns.RR {
 		v4 = s.service.AddrIPv4
 		v6 = s.service.AddrIPv6
 	}
+	if ttl > 0 {
+		// force low timeout for A/AAAA responses, as network interface
+		// up state and IPs are dynamic.
+		ttl = 120
```

Comment:
I don't understand this hack. Is it part of the standard?
Otherwise, I don't think that hardcoding the value is working for everybody using this library.

Reply:
Yes it is in the RFC: https://tools.ietf.org/html/rfc6762#section-10

>the recommended TTL value for Multicast DNS resource records with a host name as the resource record's name (e.g., A, AAAA, HINFO) or a host name contained within the resource record's rdata (e.g., SRV, reverse mapping PTR record) SHOULD be 120 seconds.

Reply:
Ok. Thanks for clarifying. 
Could you add a reference to that section in the code?